### DISCUSSION TITLE
Not enough memory on 8GB vram

### DISCUSSION THREAD

**Discussion by sithil94** [lshqqytiger/stable-diffusion-webui-directml#18] [Q&A] [answered]
Hey,
I got it working on an RX480 8GB card on windows 10 with 16GB ram. I tried different combinations of command line arguments to lower the VRAM usage but I still got out of GPU memory at 512x512. The image generation reaches 100% but crashes right when it is supposed to display the image. The VRAM usage is 7.3/8.0 in the task manager. Is it possible to allocate more memory?

I have COMMANDLINE_ARGS=--opt-sub-quad-attention-invokeai --lowvram --disable-nan-check --always-batch-cond-uncond

- While using Euler sampling, I can go up to 488x488.

- While using Euler A sampling, I can go up to 504x504. 

Other resolutions such as 512x504 and 512x520 work randomly but often gives error.
I read that the webui runs on half-precision by default, but I have used onnx FP_16 models to generate much larger images.

**Comment by morpheusseu** [comment, marked as answer]
with this `--opt-sub-quad-attention --no-half --precision full --medvram --disable-nan-check --autolaunch` I could have 800*600 with my 6600xt 8g, not sure if your 480 could make it.
but now i switch to nvidia mining card p102 10g to generate, much more effcient but cheap as well (about 30 dollar) .

**Reply by sithil94** [reply]
Unfortunately, it did not help. I saw that I get this error on cmd,

"Warning: experimental graphic memory optimization is disabled due to gpu vendor. Currently this optimization is only available for AMDGPUs.
Disabled experimental graphic memory optimizations.".

Could this be the issue?

**Reply by Miraihi** [reply]
This message shows for literally all users of SD DirectML (Me included). Because why use this version if you have Nvidia GPU and you can just download the original.

**Reply by Miraihi** [reply]
Also you'd probably also need `--opt-split-attention` or `--opt-split-attention-v1` (The second one is safer, but marginally slower).
